cypress/integration/adminFlows/pages/landingPage.spec.js

```diff
@@ -4,23 +4,28 @@ describe('Set a landing page from the admin portal', () => {
     cy.fixture('users/adminUser.json').as('user');
     cy.get('@user').then((user) => {
       cy.loginAndVisit(user, '/admin/customization/config').then(() => {
-        // Ensure Forem instance is private
-        // NOTE: @citizen428 - We may need to find a better situation for this
-        // long-term.
+        cy.get('#new_settings_user_experience').as('userExperienceSectionForm');
+        //   // Ensure Forem instance is private
+        //   // NOTE: @citizen428 - We may need to find a better situation for this
+        //   // long-term.
         cy.findByTestId('authSectionForm').as('authSectionForm');
 
-        cy.get('@authSectionForm').findByText('Authentication').click();
-        cy.get('@authSectionForm')
-          .findByLabelText('Invite-only mode')
-          .should('not.be.checked')
-          .check();
+        cy.get('@userExperienceSectionForm')
```

Comment:
Refactor (blocking): Prefer findByRole over findByText in these tests. It makes for a more reliable test. See our [E2E docs for writing tests](https://docs.forem.com/tests/e2e-tests/#writing-cypress-tests) (the Finding Elements section).

Reply:
Done!